calico/scripts/install-calico-windows.ps1

```diff
@@ -166,11 +166,30 @@ function GetBackendType()
             throw "{{site.prodname}} on Linux has IPIP enabled. IPIP is not supported on Windows nodes."
         }
 
+        # Check FelixConfig first.
         $encap=c:\k\kubectl.exe --kubeconfig="$KubeConfigPath" get felixconfigurations.crd.projectcalico.org default -o jsonpath='{.spec.vxlanEnabled}' -n $CalicoNamespace
         if ($encap -EQ "true") {
             return ("vxlan")
+        } elseif ($encap -EQ "false") {
+            return ("bgp")
+        } else {
+           # If any IPPool has IPIP enabled, we need to exit the installer. The
+           # IPIP-enabled might not be assigned to this Windows node but we can't
+           # verify that easily by looking at the nodeSelector.
+           $ipipModes = c:\k\kubectl.exe --kubeconfig="$KubeConfigPath" get ippools.crd.projectcalico.org -o jsonpath='{.items[*].spec.ipipMode}' -n $CalicoNamespace
+           $ipipEnabled = $ipipModes | Select-String -pattern '(Always)|(CrossSubnet)'
+           if ($ipipEnabled -NE $null) {
+               throw "Failed to auto detect backend type. IPIP is not supported on Windows nodes but found IP pools with IPIP enabled. Rerun install script with the CalicoBackend param provided"
+           }
+
+           # If FelixConfig does not have vxlanEnabled then check the first IPPool's vxlan mode.
+           $encap=c:\k\kubectl.exe --kubeconfig="$KubeConfigPath" get ippools.crd.projectcalico.org -o jsonpath='{.items[0].spec.vxlanMode}' -n $CalicoNamespace
```

Comment:
Shall we get all ippools with `item[*]` and grep for VXLAN? It is possible the first ippool is disabled (I think we have some use case for it) and user doesn't bother of setting correct vxlanMode.

Reply:
Yeah I think you're right. Looks like Felix determines if VXLAN is on based on whether any ippool has VXLAN enabled.